tests/functional/vault/test_misc.py

```diff
@@ -145,3 +145,16 @@ def test_deposit_withdraw_faillure(token, gov, vault):
 
     with brownie.reverts():
         vault.withdraw(vault.balanceOf(gov), {"from": gov})
+
+
+def test_report_loss(token, gov, vault, strategy, accounts):
+    strategy.harvest()
+    strategy_account = accounts.at(strategy.address, force=True)
+    token.transfer(gov, token.balanceOf(strategy.address), {"from": strategy_account})
+    assert token.balanceOf(strategy.address) == 0
```

Comment:
We added `_takeFunds` for this purpose to `TestStrategy`:
https://github.com/iearn-finance/yearn-vaults/blob/6ba0909826f7eb5a9671b9c047687c8b519f0e77/contracts/test/TestStrategy.sol#L27-L29

Reply:
Oh, got it!